src/Functions/Public/catalog-service/Get-vRAResource.ps1

```diff
@@ -116,36 +116,17 @@
                 
                     foreach ($ResourceId in $Id) { 
                 
-                        $URI = "/catalog-service/api/consumer/resourceViews/$($ResourceId)?withExtendedData=true&withOperations=true"
+                        $URI = "/catalog-service/api/consumer/resourceViews?`$filter=id eq '$($ResourceId)'&withExtendedData=true&withOperations=true"
 
                         $EscapedURI = [uri]::EscapeUriString($URI)
 
                         $Resource = Invoke-vRARestMethod -Method GET -URI $EscapedURI -Verbose:$VerbosePreference
```

Comment:
My bad @jonathanmedd , I forgot to rename $Resource to $Response here in your version of the script.